changelog/@unreleased/pr-4778.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
+improvement:
+  description: AtlasDB now allows users to pass in a `DialogueClients.ReloadingFactory`
+    in install configuration. This is likely to be useful for multitenant users of
```

Comment:
Please don't use 'multitenant' in this context, the noun is already 'taken' so it's likely to be confusing.